### DISCUSSION TITLE
Share API URLs here

### DISCUSSION THREAD

**Discussion by juliasilge** [posit-conf-2023/vetiver#32] [General]
🎉

**Comment by ldominguez50** [comment]
https://invisible-panda.d01bf.fleeting.rstd.io/rsconnect/inspections_chicago_luis_api/ (R)

**Comment by vasantm** [comment]
Try my R version: https://elastic-dingo.d01bf.fleeting.rstd.io/rsconnect/RF123-chicago/

**Comment by aaronball85** [comment]
https://indigo-seahorse.d01bf.fleeting.rstd.io/rsconnect/aaron/predict (R)

**Comment by AlaaAAlghwiri** [comment]
https://rigorous-buzzard.d01bf.fleeting.rstd.io/rsconnect/alaaalghwiri

**Reply by AlaaAAlghwiri** [reply]
This is Python

**Comment by rserran** [comment]
https://skilled-numbat.d01bf.fleeting.rstd.io/rsconnect/chicago-inspections-api/

**Comment by KoderKow** [comment]
R: https://therapeutic-horse.d01bf.fleeting.rstd.io/rsconnect/kyles-awesome-api/

**Comment by hoeoek** [comment]
R: https://unbiased-uakari.d01bf.fleeting.rstd.io/rsconnect/yay/

**Comment by dstern04** [comment]
R: https://sloppy-galago.d01bf.fleeting.rstd.io/rsconnect/dstern-random-forest/

**Comment by rkb965** [comment]
R 👿 🐦: https://demonic-caracara.d01bf.fleeting.rstd.io/rsconnect/avoid-foodborne-illness

**Comment by camcmac** [comment]
R: "https://futuristic-quokka.d01bf.fleeting.rstd.io/rsconnect/cams-mod/predict"

**Reply by rkb965** [reply]
You got such a great assigned url!

**Reply by camcmac** [reply]
right?! how did it know i lived in the home of the quokka for 5 years?

**Reply by rkb965** [reply]
oh wow, even better!

**Comment by nelliegp** [comment]
R: https://awesome-sheep.d01bf.fleeting.rstd.io/rsconnect/inspection_chicago_sheepmodel/

**Comment by jcstock** [comment]
https://cobalt-bug.d01bf.fleeting.rstd.io/rsconnect/chi-town-food-safety/

**Comment by ezraporter** [comment]
R: https://wine-petrel.d01bf.fleeting.rstd.io/rsconnect/inspections/predict

**Comment by aluxh** [comment]
Python: https://masked-toucanet.d01bf.fleeting.rstd.io/rsconnect/alex-python-inspect/
R: https://masked-toucanet.d01bf.fleeting.rstd.io/rsconnect/alex-inspect/
